docs/kubernetes/how-to-import-volumes.md

```diff
@@ -0,0 +1,41 @@
+# How to import volumes
+
+This guide explains how to import an existing Hetzner volume into your Kubernetes cluster with the csi-driver installed.
+
+1. Detach your volume by running `hcloud volume detach <volume-name>`
+2. Find the ID of your volume by running `hcloud volume list`
+3. Create a new PersistentVolume and insert the volume ID into the `volumeHandle`
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: imported-data
+spec:
```

Comment:
Could you also add the topology constraints? They are added by the csi-driver otherwise, and not having them can lead to downtime later on.